### PR TITLE
Use single api update lambda

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: lambda/api_update.tf [hunk 1]
```diff
@@ -15,37 +14,37 @@ resource "aws_lambda_function" "lambda_api_update_function" {
...
   assume_role_policy = templatefile("${path.module}/templates/lambda_assume_role.json.tpl", {})
   name               = "${upper(var.project)}ApiUpdateRole"
 }
 
 resource "aws_iam_role_policy_attachment" "lambda_api_update_role_policy" {
-  count      = local.count_api_update_av
+  count      = local.count_api_update
   policy_arn = aws_iam_policy.lambda_api_update_policy.*.arn[0]
   role       = aws_iam_role.lambda_api_update_iam_role.*.name[0]
 }
```
Comment (TomJKing) [inline lambda/api_update.tf:50]: Missing line break end of file

File: lambda/locals.tf [hunk 2]
```diff
@@ -4,14 +4,12 @@ locals {
   count_av_yara                  = var.apply_resource == true && var.lambda_yara_av == true ? 1 : 0
   count_checksum                 = var.apply_resource == true && var.lambda_checksum == true ? 1 : 0
   count_log_data                 = var.apply_resource == true && var.lambda_log_data == true ? 1 : 0
-  count_api_update_av            = var.apply_resource && var.lambda_api_update_av == true ? 1 : 0
+  count_api_update            = var.apply_resource && var.lambda_api_update == true ? 1 : 0
```
Comment (TomJKing) [inline lambda/locals.tf:7]: Did you run terraform fmt?

File: lambda/api_update.tf (hunk 1 above)
Comment (MancunianSam) [reply lambda/api_update.tf:50]: I've added it in.

File: lambda/locals.tf (hunk 2 above)
Comment (MancunianSam) [reply lambda/locals.tf:7]: No, I've run it now.

**Pull request merged**